jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml

```diff
@@ -25,10 +26,19 @@ rules:
       id: "$3"
   # "io.confluent.ksql.metrics:type=_confluent-ksql-<cluster-id>ksql-engine-query-stats"
   # The below statement parses KSQL Cluster Name and adds a new label so that per cluster data is searchable.
-  - pattern: io.confluent.ksql.metrics<type=_confluent-ksql-(.+)ksql-engine-query-stats><>([^:]+)
+  - pattern: io.confluent.ksql.metrics<type=_confluent-ksql-(.+)_ksql-engine-query-stats><>([^:]+)
     name: "ksql_ksql_engine_query_stats_$2"
     labels:
       ksql_cluster: $1
+  # "io.confluent.ksql.metrics:type=ksql-queries,status=_confluent-ksql-<cluser-id>_query_<query>
+  # The below statement parses KSQL query specific status
+  - pattern: "io.confluent.ksql.metrics<type=(.+), status=_confluent-ksql-(.+)_query_(.+)><>(.+): (.+)"
+    value: 1
+    name: $1_status
```

Comment:
Could you update it to follow the naming convention -- componentName_beanName_typeName_attributeName ?
Eg:
io.confluent.ksql.metrics:status=_confluent-ksql-ksql-clusterquery_CSAS_EN_WIKIPEDIA_GT_1_COUNTS_9,type=ksql-queries": {
      "query-status": "RUNNING",
      "error-status": "NO_ERROR"
    }
would become :
ksql_ksql_metrics_ksql_queries_query_status
ksql_ksql_metrics_ksql_queries_error_status

Reply:
updated to match the existing naming convention